.vscode/settings.json

```diff
@@ -1,7 +1,5 @@
 {
-  "editor.rulers": [
-    80
-  ],
+  "editor.rulers": [80],
```

Comment:
Can we undo this change? It's probably prettier, but this might make it harder to merge if someone else has edited this file?